libraries/AP_Common/Location.cpp

```diff
@@ -244,6 +244,16 @@ ftype Location::get_distance(const struct Location &loc2) const
     return norm(dlat, dlng) * LOCATION_SCALING_FACTOR;
 }
 
+// return the altitude difference in meters taking into account alt frame.
```

Comment:
We should be clear about which way around this is - e.g. `distance will be positive if loc2 is above *this` (I'm not even sure that's correct :-) )

Reply:
I copied the why we do it in distance NED, but as you say there both confusing.